pkg/util/fs_util.go

```diff
@@ -245,8 +245,8 @@ func extractFile(dest string, hdr *tar.Header, tr io.Reader) error {
 				return errors.Wrapf(err, "error removing %s to make way for new link", hdr.Name)
 			}
 		}
-
-		if err := os.Link(filepath.Clean(filepath.Join("/", hdr.Linkname)), path); err != nil {
+		link := filepath.Join(dest, hdr.Linkname)
+		if err := os.Link(filepath.Clean(filepath.Join("/", link)), path); err != nil {
```

Comment:
I'm curious to know why "/". `path` is essentially `filepath.Join(dest, hdr.Name)`, so it seems reasonable that the first arg to `os.Link` is `filepath.Join(dest, hdr.Linkname)`, no?

Reply:
thanks for pointing that out! i've removed the unnecessary filepath.Join